modules/vaos/config/routes.rb

```diff
@@ -2,6 +2,7 @@
 
 VAOS::Engine.routes.draw do
   namespace :v0, defaults: { format: 'json' } do
-    get 'appointments', to: 'vaos#get_appointments'
```

Comment:
This snuck in earlier even though there was no appointments endpoint yet. Another PR will soon add it back in, so... 🤷‍♂